### ISSUE TITLE
set surface info not take effect in method add_surface_points

### ISSUE THREAD

**Issue by llcat** [cgre-aachen/gempy#813]
**Describe the bug**
set surface info not take effect in method add_surface_points

**To Reproduce**
Provide detailed steps to reproduce the behavior:
1.  use getting started code 
```py
# Importing GemPy
import gempy as gp

geo_model = gp.create_model('Model1')
geo_model = gp.init_data(geo_model, extent=[0, 791, 0, 200, -582, 0], resolution=[100, 10, 100])

gp.set_interpolator(geo_model, verbose=[])

geo_model.set_default_surfaces()

geo_model.add_surface_points(X=223, Y=0.01, Z=-94, surface='surface1')
geo_model.add_surface_points(X=458, Y=0, Z=-107, surface='surface1')
geo_model.add_surface_points(X=612, Y=0, Z=-14, surface='surface1')
geo_model.add_orientations(X=350, Y=0, Z=-300, surface='surface1', pole_vector=(0, 0, 1))

geo_model.add_surface_points(X=225, Y=1, Z=-269, surface='surface2')
geo_model.add_surface_points(X=459, Y=1, Z=-279, surface='surface2')

gp.compute_model(geo_model)

# Plot in 2D
gp.plot_2d(geo_model, cell_number=5, legend='force')
# Plot in 3D
# gp.plot_3d(geo_model)
```

**Expected behavior**
the points in surface2 show correct info

**Screenshots**
points at surface2 still show surface1 color
![image](https://github.com/cgre-aachen/gempy/assets/17900822/8bca6904-ef2b-46f9-b069-906436d96ca8)
![image](https://github.com/cgre-aachen/gempy/assets/17900822/9d13ef91-1528-4992-95e7-d296f9ff0bf9)

**Desktop (please complete the following information):**
  * OS: (Win 11)
  * GemPy Version
    2.3.0
  * Jupyter Version (if applicable)

**Additional context**

**Comment by Japhiolite** [comment]
Hi @llcat 
thanks for submitting this bug. We'll take a look into it asap. Stay tuned

**Comment by Japhiolite** [comment]
Should be fixed in the new released version 2.3.1 on pip and here as release.
